dictdiffer/__init__.py

```diff
@@ -203,6 +209,9 @@ def check(key):
                 # callees again diff function to compare.
                 # otherwise, the change will be handled as `change` flag.
                 if path_limit and path_limit.path_is_limit(_node + [key]):
+                    if deepcopy(_first[key]) == deepcopy(_second[key]):
```

Comment:
Do we really need to call `deepcopy` for comparing values?

Reply:
It's not necessary.